app/assets/stylesheets/styles.scss

```diff
@@ -5,3 +38,99 @@
 .centered {
   text-align: center;
 }
+
+.goals-list {
+}
+
+.wordwrap {
+   white-space: pre-wrap;
+   white-space: -moz-pre-wrap;
+   white-space: -pre-wrap;
+   white-space: -o-pre-wrap;
+   word-wrap: break-word;
+}
+
+.goals-header {
+  text-align: center;
+  padding-bottom: 20px;
+}
+
+.add-goal {
+  padding-top: 50px;
+  padding-bottom: 20px;
+}
+
+.goal-details {
+  background-color: #ffffff;
+  font-family: Roboto, sans-serif;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, font-family, height, margin